torctl

```diff
@@ -152,7 +152,7 @@ wipe() {
 }
 
 get_ip() {
-    RADDR=$(curl -s https://ipinfo.io/ip)
+    RADDR=$(curl -s https://api.ipify.org/)
```

Comment:
```suggestion
    RADDR=$(curl -s https://checkip.amazonaws.com/)
```

I'd suggest using checkip.amazonaws.com since all the others usually run behind cloudflare or tend to block tor requests, I've had this issue for days now, the amazon one doesn't give me any issues.